mtp_cashbook/templates/cashbook/transactions_history.html

```diff
@@ -40,18 +40,45 @@ <h1 class="error-summary-heading heading-medium" id="error-summary-heading">{% t
       </div>
     {% endif %}
 
-    {% for field in form %}
-      <div id="form-group-{{ field.html_name }}" class="form-group {% if field.errors %}error{% endif %}">
-        <label for="{{ field.id_for_label }}" id="{{ field.auto_id }}-label" class="form-label">
-          {{ field.label }}
-          {% for error in field.errors %}
-            <span class="error-message">{{ error }}</span>
-          {% endfor %}
-        </label>
-        {{ field }}
+    <div class="grid-row">
+    <div id="form-group-search" class="form-group ">
+        <div class="column-full">
+          <label for="id_search" id="id_search-label" class="form-label">
+            {% trans 'Prisoner name, prisoner number or sender name' %}
```

Comment:
The problem with how it was before is that the backend generates the form, so there's no control on style. Worse, there's non-translatable text in forms.py